src/promptflow-devkit/promptflow/_sdk/operations/_flow_operations.py

```diff
@@ -498,18 +498,15 @@ def _export_flow_connections(
                 return self._migrate_connections(
                     connection_names=SubmitterHelper.get_used_connection_names(
                         tools_meta=CSharpExecutorProxy.generate_flow_tools_json(
-                            flow_file=flow.flow_dag_path,
+                            flow_file=flow.flow_file_path,
                             working_dir=flow.code,
                         ),
                         flow_dag=flow._data,
                     ),
                     output_dir=output_dir,
                 )
             else:
-                # TODO: avoid using executable here
```

Comment:
NIT: please keep this TODO given we still use executable here

Reply:
done